.github/workflows/build.yml

```diff
@@ -41,16 +41,13 @@ jobs:
         uses: ./godot-cpp/.github/actions/godot-cache
         with:
           cache-name: windows-x86_64
-      - uses: actions/setup-python@v2
-        with:
-          python-version: "3.x"
-          architecture: "x64"
```

Comment:
Removed this as Python is already default installed. One consequence was that the user pip install path was not in `PATH` anymore, so I changed the installs to be global (which is what we do for Godot too).